common/vendor/MarkdownEditor/angular-markdown-editor.js

```diff
@@ -0,0 +1,200 @@
+angular

```

Comment:
Is this directive something that you wrote yourself? Or is it a third-party plugin? If it's thirdparty include its source. If it's something that you wrote or modified add some comments and documentation on how it works.